### PR TITLE
UND-818- UI Breaking issue

### PR DESCRIPTION
**Pull request by neelesh-tippe**
[UND-818][DEVELOP]- UI Breaking issue

### CODE REVIEW

File: stories/Components/UIcomponents/Accordion/Accordion.jsx [hunk 1]
```diff
@@ -1,37 +1,62 @@
...
+        <button id={accordionButtonID2} tabIndex="0" aria-controls={accordionpanelId2} aria-expanded="false">{headerText}</button>
+        <div id={accordionpanelId2} className="accordion__panel" aria-labelledby={accordionButtonID2} aria-hidden="true" role="region">
           <P label={descriptionText} />
           <P label={descriptionText} />
         </div>
       </li>
     </ul>
   );
+
+  // return (
```
Comment (shishirphadke) [inline stories/Components/UIcomponents/Accordion/Accordion.jsx:37]: @neelesh-tippe Remove commented code.

Comment (neelesh-tippe) [reply stories/Components/UIcomponents/Accordion/Accordion.jsx]: Done

File: stories/Components/UIcomponents/Accordion/accordion.scss [hunk 2]
```diff
@@ -8,70 +8,78 @@
...
+        width: 1.25rem;
+        position: absolute;
+        top: $spacing-24;
+      }
 
-    .accordion__heading {
-      margin-bottom: 0.625rem;
-      padding-right: $spacing-24;
+      &.accordion--active {
+        padding-bottom: 0;
```
Comment (shishirphadke) [inline stories/Components/UIcomponents/Accordion/accordion.scss:47]: @neelesh-tippe Don't remove the bottom padding for the `active` li as this causes the `accordion__panel` to jump down a bit while slide up animation(when the accordion is closing). Instead reduce the top padding for `accordion__panel`. 
Eg:
```
.accordion__panel { 
  padding: $spacing-24 $spacing-32 $spacing-32;
  @include devicebreak(medium) {
    padding: $spacing-24 $spacing-40 $spacing-40;
  }
}
```

Comment (neelesh-tippe) [reply stories/Components/UIcomponents/Accordion/accordion.scss]: &.accordion--active {
        padding-bottom: 0;

 is removed from code, there were no specific use of it.

**Pull request merged**